modules/ROOT/pages/bonita-bpm-overview.adoc

```diff
@@ -1,81 +1,94 @@
+Morevoer, Bonita integrates with existing information systems, orchestrates heterogeneous systems, and provides deep visibility into processes across the organization.
 
-== Bonita Studio
-
-Bonita Studio contains all the elements needed to develop and compile a Bonita application. It is a graphical environment for creating processes, applications, data models, and users views (pages and forms). It contains three major design tools:
-
-* the whiteboard, for drawing a process flow diagram and defining the detail of steps, transitions, decision points and other process elements
-* the Development menu, to extend the Studio capabilities and create your data models
-* the UI Designer, which is used to create application pages and process forms
-
-Bonita Studio is the development tool for the Business Analyst and Application Developer. Back-end java developers work in Bonita Studio and collaborate with front-end developers working in the UI Designer.
-
-You can xref:bonita-bpm-installation-overview.adoc[install Bonita Studio] on your computer, then create and test processes, create data models, applications and application views (pages and forms).
-
-Bonita Studio contains a *Bonita Runtime* (Tomcat, UI Designer, Bonita Portal, Bonita Engine, and an h2 database), suitable for testing an application that is in development. When you run a process, it is automatically deployed onto the development platform.
+The Bonita platform has two main components: the development environment, Bonita Studio, and the runtime environment, Bonita Runtime.
 
-Process forms, used to complete the human tasks, are created in the UI Designer, and use the data models created in the Studio.
+== Bonita Studio
 
-Users can use Bonita Portal (_User_ profile) to view and complete the process tasks for all processes. You can also build personalized process-based applications. To do so, use the UI Designer to create application pages that display business data from the data models, link the pages to processes thanks to action buttons, then use the Bonita Portal application builder to construct the application piece by piece.
+Bonita Studio contains all the elements needed to design and compile processes allowing entities of an organization to handle data in a standardized and traceable way. +
+Indeed, it is a graphical environment for creating processes and data models, to manage the organization of people that will do the tasks, and create the connectivity between Bonita and the IT environment it will orchestrate. +
 
-[#platform]
+By default, users can use the Bonita User application to view and complete the process tasks for all processes. +
+But with Bonita, you can also build personalized process-based applications that will display customized views of the business data. +
 
-== Bonita Runtime
+To do so, Bonita Studio also contains an application designer, and a UI Designer.
```

Comment:
Sentence moved into the bloc above to avoid unwanted line break. Also, I suggest top avoid the  'to do so', it fells weird. And you start the next line with 'So' -> 

'To do so' ...
'So' ...

-> No pleasant to read, it feels like you are speaking.

```suggestion
```